### PR TITLE
dataforwarding-infrequent

### PR DESCRIPTION
**Pull request by JainM6**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/manage/partitions-data-tiers/infrequent-tier-data-forwarding.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  <meta name="robots" content="noindex" />
+</head>
+
+<p><a href="/docs/beta"><span className="beta">Beta</span></a></p>
+
+Previously, data forwarding was only supported in Sumo Logic’s Continuous data tier. Now, you can forward log data from a [Partition](/docs/manage/partitions-data-tiers/) or [Scheduled View](/docs/manage/scheduled-views/) in Infrequent tier. This support helps debugging of rare issues by utilizing verbose log sources, ensuring effective troubleshooting for specific scenarios that occur infrequently.
+
+## Infrequent data forwarding support
+
+To enable data forwarding in Infrequent tier, follow the steps:
```
Comment (JV0812) [inline docs/manage/partitions-data-tiers/infrequent-tier-data-forwarding.md:19]:
```suggestion
To enable data forwarding in the Infrequent tier, follow the steps:
```

File: docs/manage/partitions-data-tiers/infrequent-tier-data-forwarding.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+id: infrequent-tier-data-forwarding
+title: Infrequent Tier Support for Data Forwarding (Beta)
+description: Learn how to forward log data in Infrequent data tier.
```
Comment (JV0812) [inline docs/manage/partitions-data-tiers/infrequent-tier-data-forwarding.md:4]:
```suggestion
description: Learn how to forward log data in the Infrequent data tier.
```

**Review by JV0812 — CHANGES_REQUESTED** [review]
Minor formatting

File: docs/manage/partitions-data-tiers/infrequent-tier-data-forwarding.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+<p><a href="/docs/beta"><span className="beta">Beta</span></a></p>
+
+Previously, data forwarding was only supported in Sumo Logic’s Continuous data tier. Now, you can forward log data from a [Partition](/docs/manage/partitions-data-tiers/) or [Scheduled View](/docs/manage/scheduled-views/) in Infrequent tier. This support helps debugging of rare issues by utilizing verbose log sources, ensuring effective troubleshooting for specific scenarios that occur infrequently.
+
+## Infrequent data forwarding support
+
+To enable data forwarding in Infrequent tier, follow the steps:
+1. In Sumo Logic, go to **Manage Data** > **Logs** > **Partitions**, or **Manage Data** > **Logs** > **Scheduled Views**, depending on whether you want to forward data from a Partition or a Scheduled View.
+1. Click the Partition or View for which you want to enable data forwarding and click the **Edit** button. When editing, you'll see an option to **Enable Data Forwarding**. <br/> <img src={useBaseUrl('img/partitions-data-tiers/enable-infrequent.png')} alt="data-forwarding" width="400" />
```
[the remark this answers is not in the document]
Comment (JainM6) [reply docs/manage/partitions-data-tiers/infrequent-tier-data-forwarding.md:21]: This is correct as we are here talking in a generic way.

**Review by JV0812 — APPROVED** [review]

**Pull request merged**